python/ray/util/sgd/torch/worker_group.py

```diff
@@ -191,6 +192,19 @@ def _setup_operator(self):
         ]
         return remote_operator_setups
 
+    def _setup_local_rank(self, rank_counter_dict=defaultdict(int)):
```

Comment:
Avoid setting mutable defaults - https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
just do `rank_counter_dict = None` and then in the code, do `rank_counter_dict = rank_counter_dict or defaultdict(int)`

Reply:
changed